review-sentiment/tests/runtime_test.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import numpy as np
+import re
+import string
+import os
+from tabulate import tabulate
+
+#---------------------------------------------------------------------------------------
+# TEST ROUTINE
+
+def evaluate(review: str, nof_requests: int, path: str, bw_cloud: bool):
+    split_review = re.findall(r"[\w']+|" + f"[{string.punctuation}]", review)
+    # initialize prediction occurence counter
+    prediction_occ = np.zeros(5)
+    prediction_times = []
+    # create dict for predict request
+    predict_json = {"text": review}
+
+    # initialize explanation score lists (one list for every word/symbol)
+    explanation_scores = [[] for _ in range(len(split_review))]
+    explanation_times = []
+
+    for _ in range(nof_requests):
+        # request prediction scores
+        if bw_cloud:
+            prediction_request = requests.post(bwc_predict_link, json=predict_json)
+            # choose arg max as prediction
+            prediction = np.argmax(prediction_request.json()['prediction'])
+            prediction_times.append(prediction_request.elapsed.total_seconds())
+
+            # create dict for explain request
+            explain_json = {"text": review, "target": str(np.argmax(prediction)),
+                            "explainer": "integrated_gradients"}
+            # request explanation scores
+            explanation_request = requests.post(bwc_explain_link, json=explain_json)
+            # save duration of explain request
+        else:
+            prediction_request = requests.post(gc_predict_link, json=predict_json)
+            # choose arg max as prediction
+            print(prediction_request.json())
+            prediction = np.argmax(prediction_request.json()['prediction'])
+            prediction_times.append(prediction_request.elapsed.total_seconds())
+
+            # create dict for explain request
+            explain_json = {"text": review, "target": str(np.argmax(prediction)),
+                            "explainer": "integrated_gradients"}
+            # request explanation scores
+            explanation_request = requests.post(gc_explain_link, json=explain_json)
+        explanation_times.append(explanation_request.elapsed.total_seconds())
+
+    # print results
```

Comment:
as above, better: cloud_name = "BW Cloud" if bw_cloud else "Google Cloud" and then insert cloud_name into write/print statement